### ISSUE TITLE
What is the point of bootstrap_agent.sh

### ISSUE THREAD

**Issue by smeyerhot** [thundergolfer/example-bazel-monorepo#50]
## Description of the problem / feature request:
It looks like bootstrap_agent.sh does just about nothing. What is the point of this script, and why do you have it in a s3 bucket?

### 🐛 Bug Report: Steps to Reproduce the Problem

  1.
  2.
  3.

### 🐛 Bug Report: Specifications

- Version:
- Platform:

**Comment by thundergolfer** [comment]
If you look through the history of that file, you'll see it used to install some stuff that I hadn't baked into the AWS AMI, such as the _Black_ Python source code formatter.

Recently I removed the need to install anything, and so the file became a kind of 'no op', but the Buildkite config still expects to find and run that file, so it still exists. 

In future I could just remove the config that has Buildkite expecting to run that file, and then remove the file.

**Comment by smeyerhot** [comment]
Ok, fair enough. Is the ami is for the buildkite agent?

**Comment by thundergolfer** [comment]
Yes the AMI is for the Buildkite agent. It's not a nice dev-loop to have to rebuild AMIs, but not bothered to change it for now.
